classes/controller/AdminController.php

```diff
@@ -4733,4 +4734,46 @@ private function getTabLinkFromSubTabs(array $subtabs)
 
         return '';
     }
+
+    /**
+     * Redirect to the Symfony controller.
+     *
+     * This method can be useful for partly-migrated controllers,
+     * to prevent the user from accessing the legacy controller
+     * and redirect to the Symfony controller instead.
+     * Forwards the messages (if any) as well.
+     */
+    protected function redirectToSymfonyControllerWithMessages()
```

Comment:
is it worth dealing with legacy part? the end goal is to remove it, so we can migrate CRUD in another Pr and only then enable new page maybe?